src/request-message.js

```diff
@@ -0,0 +1,22 @@
+import {join, buildQueryString} from 'aurelia-path';
+import {Headers} from './headers';
+
+export class RequestMessage {
+  constructor(method, url, content, headers) {
+    this.method = method;
+    this.url = url;
+    this.content = content;
+    this.headers = headers || new Headers();
+  }
+
+  fullUrl() {
```

Comment:
I think the old name of `buildFullUrl` is clearer because the verb clearly indicates what the method does.